referrer-policy/generic/template/test.html.template

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+%(generated_disclaimer)s
+<html>
+  <head>
+    <title>Referrer-Policy: %(spec_title)s</title>
+    <meta name="description" content="%(spec_description)s">
+    %(meta_delivery_method)s
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <!-- TODO(kristijanburnik): This is not needed for all tests, only for fetch. -->
+    <script src="/referrer-policy/generic/fetch.js"></script>
+    <!-- Common global functions for referrer-policy tests. -->
+    <script src="/referrer-policy/generic/common.js"></script>
+    <!-- The original specification JSON for validating the scenario. -->
+    <script src="/referrer-policy/spec_json.js"></script>
+    <!-- Simple wrapper API for all referrer-policy test cases. -->
+    <script src="/referrer-policy/generic/referrer-policy-test-case.js?pipe=sub"></script>
+  </head>
+  <body>
+    <h1>%(spec_title)s</h1>
```

Comment:
See http://testthewebforward.org/docs/test-templates.html#script-test-with-metadata for the recommended test format.